### DISCUSSION TITLE
GitHub mobile has a slight UI change that I would think would be nice for the product, and that would be updating the GitHub username on mobile when it’s updated on the site.

### DISCUSSION THREAD

**Discussion by DEV-DIBSTER** [community/community#63827] [Mobile]
![image](https://github.com/community/community/assets/76603072/dbb3fbda-35fb-4cf7-a489-15c6891caa6d)![image](https://github.com/community/community/assets/76603072/4c758c21-ced8-4300-ab0d-be8165b58aad)

Pictured above is this account, but if you're on the web you can see that my account username is DEV-DIBSTER instead of DIBSTERYT. Just a slight thing I noticed when using the app.
<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.126.0 (107057637)
Device: iPhone SE 2 (iOS 16.6)
</details>

**Comment by eliperkins** [comment]
Hey @DEV-DIBSTER , thanks for this feedback!

Not seeing your account's username update does sound frustrating. Can I get a little bit of information from you to help debug?

- If you quit the app and reopen it, does your username update?
- Are there other places within the app besides the account switcher where you see the old username instead?

Thanks for helping us improve the quality of GitHub Mobile!

**Comment by DEV-DIBSTER** [comment]
Hey @eliperkins, I can provide some more information.

Yeah of course. Attached below should be a screen recording of me loading up the app for the first time today, as well as then closing the app, and reopening it. 

My profile does show my updated username, as well as most everywhere else, it is only the account switcher and widgets changer which I will also attached a screenshot of.

https://github.com/community/community/assets/76603072/f5691d1d-6ebc-445e-8782-e769c8245c8b

**Reply by DEV-DIBSTER** [reply]
![image](https://github.com/community/community/assets/76603072/4af3f73c-5f79-4d08-a7cc-6a277c3dc2be)

**Reply by DEV-DIBSTER** [reply]
I also just realized when I try to reply to this, it has my old username as well.![image](https://github.com/community/community/assets/76603072/8fceb334-168a-4908-9e21-6a1a297b050d)

**Reply by eliperkins** [reply]
👍  awesome! This is super helpful info. I'll go dive into this and get this bug fixed. Thanks for the report!

**Reply by DEV-DIBSTER** [reply]
No problem! Just reply to this thread/conversation if you want additional information for debugging. 🙂
